### PR TITLE
Adds Idris Unathi

### PR DESCRIPTION
**Pull request by Haydizzle**
Title. Gives unathi the ability to play as Idris contractors with history on the shadier side of the company dealings.

### CODE REVIEW

**Comment by Haydizzle** [comment]
!review

**Review by Geevies — APPROVED** [review]

**Review by Chada-1 — APPROVED** [review]

File: html/changelogs/haydizzle-idrisnathi.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (Geevies) [inline html/changelogs/haydizzle-idrisnathi.yml:1]:
```suggestion
```

**Comment by Haydizzle** [comment]
Need a DNM on this until lore makes a decision.

**Comment by Haydizzle** [comment]
Closing after feedback from lore.

**Pull request closed without merging**